sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -201,19 +203,18 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
         "disbursementValue.valueAmount",
         "disbursement.coeUpdatedAt",
         "studentAssessment.id",
-        "studentAssessment.assessmentData",
       ])
       .innerJoin("disbursement.studentAssessment", "studentAssessment")
       .innerJoin("studentAssessment.application", "application")
       .innerJoin("application.location", "location")
-      .innerJoin("application.offering", "offering")
+      .innerJoin("application.currentAssessment", "currentAssessment") // ! This is to fetch the current assessment of the application, even though we have multiple reassessments
```

Comment:
FYI- I think `// !` is for alerts, to highlight the comment you can also use `* *` or `// *`